BedrockServer.cpp

```diff
@@ -1794,6 +1808,8 @@ void BedrockServer::_control(unique_ptr<BedrockCommand>& command) {
             response["previousMaxConflictRetries"] = to_string(_maxConflictRetries.load());
             _maxConflictRetries.store(maxConflictRetries);
         }
+    } else if (SIEquals(command->request.methodLine, "SetConflictPageLocks")) {
```

Comment:
NAB, we could make this make the `500` configurable

Reply:
We could. It's not super hard to add later but I don't think it's very important right now.